docs/plug-ins/open-source-adt-plugins.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: page
+title: Nützliche Open Source ADT Plugins
+parent: Kapitel 7 - Plug-ins
+permalink: /plug-ins/open-source-adt-plugins/
+nav_order: 2
+---
+
+1. TOC
+{:toc}
```

Comment:
I would use the toc after the main heading `# Nützliche Open Source ADT Plugins`
You can exclude the main heading with `{: .no_toc }`
https://just-the-docs.github.io/just-the-docs/docs/navigation-structure/#in-page-navigation-with-table-of-contents

Reply:
Good Idea. I updated the pull request with the requested changes

Reply:
Looks good, would be perfect if the other chapters use this as a template and then are built similar